kiwi/boot/functions.sh

```diff
@@ -5517,6 +5517,21 @@ function killShell {
     fi
 }
 #======================================
+# getDeviceTransportType
+#--------------------------------------
+function getDeviceTransportType {
+    # /.../
+    # function to get the transport type of the
+    # given device. It uses lsblk command.
+    # ----
+    local device=$1
+    if [ -z "$device" ] || [ ! -e "$device" ];then
+        echo 1 ; return 1;
+    fi
+    local disk=${device%%[0-9]*}
+    echo "`lsblk -dno tran $disk`"
```

Comment:
@isbm that's cool, I was not aware of the possibility to use arithmetic in such a way :+1:  I though it was mostly a matter of getting rid of graveticks mixed with simple or double quotes, which can be tricky.

thanks :wink: